packages/frontend-core/src/components/grid/stores/datasource.js

```diff
@@ -147,19 +168,34 @@ export const createActions = context => {
   }
 
   // Adds a schema mutation for a single field
-  const addSchemaMutation = (field, mutation) => {
+  const addSchemaMutation = (field, mutation, fromNestedField) => {
```

Comment:
I like the nice logical separation of `schemaMutations` and `subSchemaMutations`. Could we add a dedicated function for `addSubSchemaMutation` instead of keeping both types of mutation inside `addSchemaMutation`?

Reply:
Done